### PR TITLE
MD-806: update README.md file with recent change

### PR DESCRIPTION
**Pull request by cchang738**
@Agirish Ahbishek, please review. Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,16 +95,16 @@ Refer to [Github documentation](https://help.github.com/articles/cloning-a-repos
 </code></pre>
 
 ### Build test framework
-In the `framework` directory, execute `mvn clean install` first, to build the project and also download any dependent datasets configured in `pom.xml`
+In the root directory of you repository, issue `bin/build_framework` to build the project and also download any dependent datasets configured in `pom.xml`
 
 ### Execute tests
-In the `framework` directory, execute the following command to run tests:
+In the root direcotry of your repository, execute the following command to run tests:
```
Comment (Agirish) [inline README.md]: Typo.

File: README.md [hunk 2]
```diff
@@ -42,7 +45,7 @@ Refer to [Github documentation](https://help.github.com/articles/cloning-a-repos
  4. You could generate expected result files using Postgres or any such database.
  5. In the `framework/resources/Datasources` directory, create corresponding datasource directories and copy over any required scripts and datasets required by the tests.
 
-#### Structure of test definiton files
+#### Structure of test definiton files. A test definition file defines one or a set of tests in json format. The file name must ends with .json. Framework looks for files with .json extension in specified location(s) and pick up all test(s) defined by them.
```
Comment (Agirish) [inline README.md]: Can you please separate this into two lines. 

#### Structure of test definition files. 
A test definition file is a JSON file that defines a test suite, which is a set of tests within a directory. The framework scans for files with .json extension in the specified location(s) and executes all test(s) defined by it.

File: README.md [hunk 3]
```diff
@@ -18,10 +18,10 @@ The framework is built for regression, integration & sanity testing. Includes te
...
- 2. The test framework expects Drill services to be setup on a clustered environment. Refer to [Drill documentation](http://drill.apache.org/docs/installing-drill-in-distributed-mode) for details on how to setup Drill.
- 3. Copy the `drillTestConfig` file from `framework/src/main/resources` to your home directory as `~/.drillTestConfig`. Edit suitably, following instructions in the file.
- 4. Source `.drillTestConfig`. Confirm `DRILL_HOME` and other required environment variables are set. 
+ 1. The test framework requires a distributed file system such as HDFS or MapR-FS to be configured. Some of the tests can also be run against a local file system. By default, it's configured to run against MapR-FS. You can change the default behavior by modifying conf/core-site.xml. Refer to conf/core-site.xml.example for settings.
+ 2. To run all tests, Hive and HBase needs to be installed and running. To exclude Hive and HBase tests, please refer to the example in the Execute Tests section.
+ 3. The test framework should be run on a Drill cluster node. Refer to [Drill documentation](http://drill.apache.org/docs/installing-drill-in-distributed-mode) for details on how to setup Drill.
```
Comment (Agirish) [inline README.md]: This is inaccurate. The original statement above is valid. The test framework can be configured to run on a client node or on one of the cluster nodes where Drill is configured.

File: README.md (hunk 2 above)
Comment (cchang738) [inline README.md]: Separated into two lines. "that defines a test suite, which is a set of tests within a directory. " This is inaccurate so I modified it a little. Framework does not have the concept of test suite.

File: README.md (hunk 3 above)
Comment (cchang738) [inline README.md]: Added a line saying it can also be run on a client node with additional configuration.

**Pull request merged**